atomix/cluster/src/main/java/io/atomix/raft/impl/DefaultRaftServer.java

```diff
@@ -144,19 +139,6 @@ public CompletableFuture<Void> shutdown() {
     return future;
   }
 
-  @Override
-  public CompletableFuture<Void> goInactive() {
```

Comment:
I assume this was not used anywhere.

Reply:
If not, I would hope our tests would catch it :smile: